WonderMake/Utilities/Vector.inl

```diff
@@ -169,6 +169,44 @@ template<typename TRep, u32 TSize>
 	return vec;
 }
 
+template <typename TRep, u32 TSize>
+constexpr void SVector<TRep, TSize>::Floor() noexcept
+{
+	for (u32 i = 0; i < TSize; i++)
+	{
+		(*this)[i] = std::floorf((*this)[i]);
+	}
+}
+
+template <typename TRep, u32 TSize>
+constexpr SVector<TRep, TSize> SVector<TRep, TSize>::GetFloored() const noexcept
+{
+	SVector<TRep, TSize> vec = *this;
+
+	vec.Floor();
+
+	return vec;
+}
+
+template <typename TRep, u32 TSize>
+constexpr void SVector<TRep, TSize>::Round() noexcept
+{
+	for (u32 i = 0; i < TSize; i++)
+	{
+		(*this)[i] = std::roundf((*this)[i]);
+	}
+}
+
+template <typename TRep, u32 TSize>
+constexpr SVector<TRep, TSize> SVector<TRep, TSize>::GetRounded() const noexcept
+{
+	SVector<TRep, TSize> vec = *this;
+
+	vec.Round();
+
+	return vec;
+}
+
```

Comment:
Could you put these in the header file instead, I have a commit that removes this file.